sdk/deviceupdate/azure-iot-deviceupdate/src/samples/java/com/azure/iot/deviceupdate/generated/DeviceManagementCancelOrRetryDeployment.java

```diff
@@ -9,18 +9,23 @@
 import com.azure.core.util.BinaryData;
 import com.azure.identity.DefaultAzureCredentialBuilder;
 import com.azure.iot.deviceupdate.DeviceManagementClient;
-import com.azure.iot.deviceupdate.DeviceUpdateClientBuilder;
+import com.azure.iot.deviceupdate.DeviceManagementClientBuilder;
 
 public class DeviceManagementCancelOrRetryDeployment {
     public static void main(String[] args) {
-        DeviceManagementClient client =
-                new DeviceUpdateClientBuilder()
+        // BEGIN:
+        // com.azure.iot.deviceupdate.generated.devicemanagementstopdeployment.devicemanagementcancelorretrydeployment
```

Comment:
I am not sure this will inject the codesnippet correctly. I think both `BEGIN` and the tagname have to be on the same line. Same with `END` below. Could you please verify that the javadocs are generated with this codesnippet correctly?

Reply:
Ah, the line is too long and it get broken by the javaformat...

One solution might be to limit the length of the tagname.
Changing the javaformat for this particular case in a particular folder seems harder.